src/components/accounts/RecoverAccountSeedPhraseForm.js

```diff
@@ -17,11 +17,12 @@ const RecoverAccountSeedPhraseForm = ({
     checkAvailability,
     requestStatus,
     clearRequestStatus,
-    setFormLoader
+    setFormLoader,
```

Comment:
Any reason to not use just global loader status here? Is this because of account ID verification?

Reply:
Yes, it's for showing `SENDING...` properly, loader was appearing in the button while typing accountId.

Reply:
@marcinbodnar should we make it easy to create actions that opt-out of loader? Or just not use Redux action for transient stuff like looking up account existence?  We can be passing some flag in action `meta` field indicating this.

Reply:
> We can be passing some flag in action meta field indicating this.
That is a good idea, I will implement it for creating account.